tests/phpunit/CRM/Core/Payment/PayPalPNTest.php

```diff
@@ -76,8 +76,8 @@ public function tearDown() {
   public function testInvoiceSentOnIPNPaymentSuccess() {
     $this->enableTaxAndInvoicing();
 
-    $pendingStatusID = CRM_Core_OptionGroup::getValue('contribution_status', 'Pending', 'name');
-    $completedStatusID = CRM_Core_OptionGroup::getValue('contribution_status', 'Completed', 'name');
+    $pendingStatusID = CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status_id', 'Pending');
```

Comment:
not sure why this was not failing the tests on jenkins. It did locally

Reply:
hmmm strange